### ISSUE TITLE
Error on Full Disk Encryption documentation

### ISSUE THREAD

**Issue by frmoronari** [void-linux/void-docs#533]
On doc for FDE, in pass to init Luks container, is mentioned to init /dev/sda1, but this is EFI partition.
Correctly is /dev/sda2

This image show the error:
https://i.ibb.co/P6MSywY/image-2020-12-11-19-02-12.png

Another that the partition needs type 8309 (Luks) and not 8300 (Linux filesystem).

**Comment by abenson** [comment]
All of the examples are shown using Legacy BIOS Boot, not UEFI.   We could include examples for both, I guess, but that'd be repetitive.

**Comment by FrostKnight** [comment]
Yeah, can you have two separate full disk encryption guides for encryption, one for legacy and one for uefi?
And at some point one for aarch64, if its possible?

**Comment by camoz** [comment]
What about substituting `/dev/sda1` by a generic `<cryptroot>` placeholder? That also makes sure people think a bit about what they will format and don't blindly copy instructions that involve `/dev/sda*`...
